packages/patternfly-4/react-core/src/experimental/components/OverflowMenu/OverflowMenuContent.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/OverflowMenu/overflow-menu';
+import { global_breakpoint_md, global_breakpoint_lg, global_breakpoint_xl } from '@patternfly/react-tokens/dist/js';
+import { debounce } from '../../../helpers/util';
+import { OverflowMenuContext, OverflowMenuContentContext } from './OverflowMenuConstants';
+
+export interface OverflowMenuContentProps extends React.HTMLProps<HTMLDivElement> {
+  /** Any elements that can be rendered in the menu */
+  children?: any;
+  /** Additional classes added to the OverflowMenuContent */
+  className?: string;
+}
+
+export interface OverflowMenuContentState extends React.HTMLProps<HTMLDivElement> {
+  isHidden: boolean;
+  breakpoints: { 
+    [index: string]: {value: string}
+  };
+}
+
+export class OverflowMenuContent extends React.Component<OverflowMenuContentProps, OverflowMenuContentState> {
+  constructor(props: OverflowMenuContentProps) {
+    super(props);
+    this.state = {
+      isHidden: false,
+      breakpoints: {
+        global_breakpoint_md,
```

Comment:
would be a little cleaner to give the key and then below can reference with `breakpoints[breakpoint]`
```suggestion
        md: global_breakpoint_md,
```